lib/rubocop/cop/rails/find_by.rb

```diff
@@ -23,6 +23,7 @@ class FindBy < Cop
 
         def on_send(node)
           return unless where_first?(node)
+          return if where_with_method_as_param?(node)
```

Comment:
I was thinking more along the lines of one method to extract the `#where` child (if any) and one to check the arguments. 🙂